executor/runtime/docker/docker_ssh.go

```diff
@@ -135,21 +123,12 @@ func addContainerSSHDConfigWithData(c runtimeTypes.Container, tw *tar.Writer, cf
 		return err
 	}
 
-	// The format that is used for SSH Users is:
-	// $(unix username):$(app name):$(aws account id):$(task id)
-
+	containerEnv := c.Env()["NETFLIX_ENVIRONMENT"]
```

Comment:
Are you ok with this defaulting to "" if we don't have this variable?

If not, can you add a log line if !ok to say "SSH will not work!"?

Reply:
Yea, let me add an update here. I'll add something similar to the EC2 script which logs and just leaves the emergency principal.